README.md

```diff
@@ -151,6 +151,17 @@ dotProp.set(obj, 'foo.$end', 'platin-unicorn')
 
 ```
 
+### setMany
+
+Set multiple nested values by passing a map of path: value
+
+```javascript
+// Setter
+var obj = { foo: { bar: 'a' } };
+// var obj2 = dotProp.set(dotProp.set(obj, 'foo.bar', 'b') , 'foo.baz', 'x');
+var obj2 = dotProp.setMany(obj, { 'foo.bar': 'b', 'foo.baz': 'x' });
```

Comment:
Looks like we need a many group, I would expect: `setMany`, `getMany` and `deleteMany`. 
In that case I would expect an array as input and output for `getMany` and not an object. Therefore I also expect an array as input for `setMany`.